app/views/posts/show.html.erb

```diff
@@ -1,17 +1,29 @@
 <div class="post">
-<h1><%= @post.title %></h1>
-<p><%= @post.text %></p>
-<p>Likes: <%= @post.likes_counter %></p>
-<p>Comments: <%= @post.comments_counter %></p>
+  <h1><%= @post.title %></h1>
+  <p><%= @post.text %></p>
+  <p>Likes: <%= @post.likes_counter %></p>
+  <p>Comments: <%= @post.comments_counter %></p>
 </div>
-<div>
-          <ul>
-            <% @post.comments.limit(10).each do |comment| %>
-<li><%= comment.author.name %> : <%= comment.text %></li>
-            <% end %>
-          </ul>
-        </div>
+
+<div class="post-comment">
+  <%= form_with url: user_post_comments_path(user_id: @user.id, post_id: @post.id), local: true do |form| %>
+    <%= form.text_field :text, name: 'comment[text]' %>
+    <%= hidden_field_tag :authenticity_token, form_authenticity_token %>
+    <%= form.submit "Add Comment" %>
+  <% end %>
+
+  <%= button_to like_user_post_path(@post.id), method: :post, remote: true, id: 'like-button' do %>
+    Like
+  <% end %>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/forms_project_v1_1.md#:~:text=Allow%20Users%20to%20add%20likes%20to%20Posts.), as requested by the last reviewer, this is not the correct way to add a like (a method inside posts controller). Kindly add a `likes_controller.rb` inside the `app/controllers` directory and add only a `create` method in it and also add the corresponding route to it when you run `rails routes` you suppose to have a route like this :
```ruby
   POST   /posts/:post_id/likes(.:format)     likes#create
```
[Hint] your likes_controller suppose to look like this:
```ruby
class LikesController < ApplicationController
    ...

    def create
      @post = Post.find(params[:post_id])
      @like = @current_user.likes.new(post_id: @post.id)

      respond_to do |format|
        if @like.save
          format.html { redirect_to @post }
        else
          format.html { redirect_to @post }
        end
      end
    end
	  ...

  end
```

### ⚠️Important NOTICE⚠️
### Code Reviews are a part of the learning process inside Microverse and the job of the code reviewer is to make sure that you applied what you learned correctly in your project milestones, So please take the reviewer's required changes and optional suggestions as one of your learning sources.

Reply:
Dear friend, I have fully met the project requirements, as you can find on the link you provided. I insist on this not because I don't want to enjoy making changes, but because I insist on changing the task description and having clear instructions on what needs to be fulfilled. Nowhere does it say that I cannot use JavaScript to create likes. In fact, that approach aligns with Rails conventions because the code will be executed in the user's browser, not on the server. I have also fulfilled the task related to comments because it was not explicitly stated how to do it. It states that I should use the current user, which is exactly what I'm doing. I will go through all these weekly projects from the beginning over the weekend because it is impossible to learn how to create forms in Rails in just an hour, considering the time allotted to us in the program.

Reply:
sorry if the project requirements didn't specify the details that Code Reviewers suggested to you.
You can create an issue in the Microverse curriculum and mention what you need there in detail and don't forget to mention the project milestone title.